src/actions/articleActions/articleAction.js

```diff
@@ -0,0 +1,54 @@
+// import axios from "axios";
```

Comment:
Remove commented code everywhere

Reply:
All the commented out code has been removed